ProjectProposal.md

```diff
@@ -0,0 +1,27 @@
+# What are we building
+We are building a simple web-app that allows users to maintain an online task list. 
+
+## Main features
+- Create, read, update, and delete tasks from the web interface
```

Comment:
Thank you @listonjesse . An initial idea here, shall we work with lists before work with tasks. 
'create, update, delete lists -> create, read, update tasks'

Reply:
In my opinion, lists are covered in the stretch goals under "Organize tasks into groups and hierarchies". We can change the word from group to list if that makes more sense.

I think starting with all tasks on one page (just as if it were one list) will make the v1 implementation easier. We can add the ability to create multiple lists of tasks after we finish implementing tasks. It is usually easier to add structure and grouping to objects when the objects already exist.

With lists though, we should remember to add the ability to order tasks within a list. We can add that to the stretch goal